### DISCUSSION TITLE
Multi step wizard with back and next buttons?

### DISCUSSION THREAD

**Discussion by zasuh** [statelyai/xstate#3758] [Q&A] [answered]
Hello! 

I was curious whether or not someone has an example of a wizard app where each step has a form, a user can go to the next step, but if they wish to go back to change something they could?

**Comment by davidkpiano** [comment, marked as answer]
Here's a WIP demo of how you can use XState along with a router (in this case, React Router) to accomplish this: https://stackblitz.com/edit/github-sjupbu?file=src%2FApp.tsx
